Makefile

```diff
@@ -1,24 +1,4 @@
 SPHINX_BUILDARGS=
-# Note that these are keys generated by the docker rsconnect service, so are
-# not really secrets. They are saved to json to make it easy to use rsconnect
-# as multiple users from the tests
-RSC_API_KEYS=pins/tests/rsconnect_api_keys.json
-
-dev: pins/tests/rsconnect_api_keys.json
-
-dev-start:
```

Comment:
Is there a way to set up `with-connect` locally we can add in the Makefile? like `with-connect -- xyz`, even if it makes assumptions about having a `.lic` file in a certain location or other creds.

Reply:
Yeah, there's just enough complexity in setup that having a `make` target would be very helpful. Thanks!